src/box/iproto.cc

```diff
@@ -155,6 +155,7 @@ struct iproto_thread {
 
 static struct iproto_thread *iproto_threads;
 static int iproto_threads_count;
+static struct evio_service binary;
```

Comment:
Could you please put a description of this variable (what it is and why it is global)?